sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
> So I don't see any bad practice at using HTTPS FWIW.

I'm not saying that using https is bad practice. I'm saying that what is bad practice is to ignore download redirector services when they are available. That's quite different.